src/main/java/com/orange/main/products/bo/ProductDTO.java

```diff
@@ -0,0 +1,75 @@
+package com.orange.main.products.bo;
+
+import java.math.BigDecimal;
+
+
+public class ProductDTO {
+
+    public ProductDTO(Products products, String distributor, BigDecimal price, BigDecimal discount, String path){
+        this.products = products;
+        if(null != this.products.getPrices()){
+            this.products.setPrices(null);
+        }
```

Comment:
This logic should not be belonged to this class. Handling it with a method of product instead.